GCTC-NTGC/gc-digital-talent
# Feature - search page pool cards

Resolves #4291 

## Notes
- Adds multiple pool cards to Search page. User should be able to react candidates from multiple pools if they have at least 1 candidate. 

## Testing
- Go to [search page](http://localhost:8000/en/search) and test out filters seeing changes to pool cards at the bottom of the page.
- Select a pool card and sent to [Request page](http://localhost:8000/en/search/request) with the correct filters chosen and correct number of candidates.
- Submit request and check that request exists on the [admin side](http://localhost:8000/en/admin/talent-requests) and DB.
